chingu-coders/Voyage2-Turtles-02
# Master

## Purpose
Add validator, edit styling

## Testing
* Checkout pull (sorry about the messy attempt to squash commits)
* Test that HTML validator shows up
* Enter test code to see errors

## Notes
Still need to factor results to add styling. Still need to add CSS.

Failed miserably trying to squash commits LOL so before accepting the pull request please choose the option to squash the commits to one.
